core/tests/protractor_utils/StoryEditorPage.js

```diff
@@ -505,6 +505,8 @@ var StoryEditorPage = function() {
     await waitFor.visibilityOf(
       warningIndicator, 'Warning Indicator taking too long to appear.');
     await browser.actions().mouseMove(warningIndicator).perform();
+    await waitFor.countToBeGreaterThanZero(
+      warningTextElements, 'Warning Text Elements taking too long to appear');
```

Comment:
In my investigation I found out that the `warningTextElements` was counted right after the mouse moved to `warningIndicator ` i.e it didn't wait for the warningTextElements to appear first.